### PR TITLE
Add support for Capture, Credit, Void, Refund & FetchTransaction

### PR DESCRIPTION
**Pull request by frankverhoeven**

### CODE REVIEW

**Review by Feijs — APPROVED** [review]
Approved, but would like to verify the integrated functionality before merging

File: src/Message/CaptureRequest.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+{
+    /**
+     * @var string
+     */
+    protected $endpoint = 'https://www.sisow.nl/Sisow/iDeal/RestHandler.ashx/InvoiceRequest';
+
+    /**
+     * @inheritDoc
+     */
+    protected function generateSignature()
```
Comment (Feijs) [inline src/Message/CaptureRequest.php:20]: Protected functions should go after public

Comment (frankverhoeven) [reply src/Message/CaptureRequest.php:20]: I followed `AbstractRequest` which has it on top, but I could change it if you like.

Comment (Feijs) [reply src/Message/CaptureRequest.php:20]: nvm, that is SF codestyle while this follows PSR-2

**Pull request merged**